docs/reference/ambassador-with-aws.md

```diff
@@ -55,14 +55,19 @@ AWS provides three types of load balancers:
 
 In Kubernetes, when using the AWS integration and a service of type `LoadBalancer`, the only types of load balancers that can be created are ELBs and NLBs (in Kubernetes 1.9 and later).
 
-If you are running an ELB in L4 mode, you need to:
+#### L4 Load Balancer
 
-* Run the ELB with the following listener configuration:
-  * `:443` -> `:8443` (the Envoy port doesn't matter)
-  * `:80` -> `:8080` (the Envoy port doesn't matter)
-  * Configure `redirect_cleartext_from` to redirect traffic on `8080` to the secure port
+If you are running an ELB in L4 mode, then to redirect all incoming HTTP requests to HTTPS, you will need to listen on 2 ports, one for HTTP which redirects all traffic to HTTPS, and another for HTTPS.
 
```

Comment:
Grammar nit: too many commas. Make it two sentences.

When running an ELB in L4 mode, you will need to listen on two ports to redirect all incoming HTTP requests to HTTPS. The first port will listen for HTTP traffic to redirect to HTTPS, while the second port will listen for HTTPS traffic.

Reply:
fixed